CLAUDE.md

```
@@ -0,0 +1,163 @@
+# Run specific tests
+./gradlew test --tests "com.codepick.*"
+
+# Kill processes using port 8080 (if port conflict occurs)
+lsof -i :8080 | grep LISTEN | awk '{print $2}' | xargs kill -9
+```
+
+## Architecture Overview
+
+CodePick is a coding test learning automation platform built with **Event-Driven Architecture (EDA)**.
+
+### Core Architecture Principles
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)